client/sites/components/sites-dataviews/dataview-style.scss

```diff
@@ -46,3 +46,23 @@
 		}
 	}
 }
+
+// Apply consistent 24px paddings to dataviews.
+.wpcom-site:has(.is-global-sidebar-visible) .dataviews-wrapper {
```

Comment:
The DataViews in v1 aren't wrapped in a Card, so we’ll need to manually apply the styles to enforce 24px padding.